notatrix/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+        "allowJs": true,
```

Comment:
Normally, the compiler will yell if you mix `*.ts` and `*.js` files.  Since we want to migrate incrementally, we'll want to enable this flag (for now).